### PR TITLE
Styling updates to make UI more responsive in the missions pages incl…

### PR DESCRIPTION
**Pull request by dakota002**
…uding word-breaks on links, image scaling, and horizontal scrolling on tables

### CODE REVIEW

File: app/routes/missions.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { NavLink, Outlet } from '@remix-run/react'
 export default function Missions() {
   return (
     <div className="grid-row grid-gap">
-      <div className="desktop:grid-col-4">
+      <div className="desktop:grid-col-4 ">
```
Comment (lpsinger) [inline app/routes/missions.tsx:15]: Unnecessary whitespace change.

**Review by lpsinger — CHANGES_REQUESTED** [review]
What about https://designsystem.digital.gov/components/table/#responsive-stacked-table for the tables?

**Comment by dakota002** [comment]
Fixed the whitespace issue, do we want to use those stacked tables? I don't see any immediate benefit, but I am not opposed to using them

**Comment by dakota002** [comment]
That's doable, but is there any reason we are keeping these tables as markdown? We will need to define the tables as elements anyway to add the uswds utility class thing. Just thinking the files are looking less and less like markdown. Are you cool with them being turned into tsx files instead? They feel repetitive enough that we might be able to break them into components or something

**Comment by lpsinger** [comment]
> That's doable, but is there any reason we are keeping these tables as markdown? We will need to define the tables as elements anyway to add the uswds utility class thing. Just thinking the files are looking less and less like markdown. Are you cool with them being turned into tsx files instead? They feel repetitive enough that we might be able to break them into components or something

These particular files are maintained by scientists who can write Markdown but probably are not comfortable with React.

Note that we can globally set all tables in markdown to be responsive by editing the rehype settings in remix.config.js.

**Comment by dakota002** [comment]
Ah good to know, in trying that approach I think it makes some of the tables look good, albeit somewhat like nutrition fact labels
image
vs 
image

But others look a little off, with the empty cells such as this 
image
vs
image

**Comment by lpsinger** [comment]
😆 

The ones that look funky, have some colspans and rowspans missing.

**Comment by dakota002** [comment]
haha yep playing with some colspans now, is there a rehype-correct way to do colspans?

**Comment by lpsinger** [comment]
It looks like there is a mix of the two table formats. Is that intended?

Screen Shot 2022-07-18 at 16 39 14

**Comment by lpsinger** [comment]
Yeah, the tables are all responsive now, but there is something funny going on with the last row of many of the tables. For example:
Screen Shot 2022-07-19 at 11 03 35

File: app/routes/missions/fermi.md [hunk 2]
```diff
@@ -30,20 +30,19 @@ https://fermi.gsfc.nasa.gov/ssc/data/
 
 **GCN Notice Types in GCN Classic and GCN Classic Over Kafka:**
 [Detailed Descriptions and Examples](https://gcn.gsfc.nasa.gov/fermi.html)
-
-| Type                  | Contents                                              | Latency        |
+| Type | Contents | Latency |
```
Comment (lpsinger) [inline app/routes/missions/fermi.md:33]: Undo unnecessary whitespace changes here.

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — APPROVED** [review]

**Pull request merged**